rafx-assets/src/assets/graphics_pipeline/assets.rs

```diff
@@ -349,6 +350,51 @@ impl MaterialAsset {
     ) -> Option<usize> {
         self.inner.pass_phase_to_index.get(&index).copied()
     }
+
+    pub fn get_single_material_pass(&self) -> RafxResult<ResourceArc<MaterialPassResource>> {
+        if self.inner.passes.len() == 1 {
+            Ok(self
+                .inner
+                .passes
+                .get(0)
+                .unwrap()
+                .material_pass_resource
+                .clone())
+        } else {
+            Err(RafxError::StringError(
```

Comment:
This could probably be Err("Found more than one MaterialPass in MaterialAsset.")?

I'd mention "get_single_material_pass" in the error. Something like "get_single_material_pass was called on a MaterialAsset that does not have exactly 1 MaterialPass"